src/components/Content/CategoriesTable.css

```diff
@@ -1,10 +1,9 @@
 @import url('../Helper/Helper.css');
 
-.thumbnail {
+.categoryThumbnail {
```

Comment:
should use kebab-case formatting like: .category-thumbnail {

post-css takes care of making it available as richLayout to other modules.

Reply:
done and also the rest of the kebab-case formattings